openedx/features/course_experience/templates/course_experience/course-outline-fragment.html

```diff
@@ -5,121 +5,159 @@
+                                                    % endif
                                                 % endif
+                                            </span>
                                             % endif
-                                        </span>
-                                        % endif
-                                    </div> <!-- /details -->
-                                </div> <!-- /subsection-text -->
-                                <div class="subsection-actions">
-                                    ## Resume button (if last visited section)
-                                    % if subsection['current']:
-                                        <span class="sr-only">${ _("This is your last visited course section.") }</span>
-                                        <span class="resume-right">
-                                            <b>${ _("Resume Course") }</b>
-                                            <span class="icon fa fa-arrow-circle-right" aria-hidden="true"></span>
-                                        </span>
-                                    %endif
-                                </div>
-                            </a>
-                        </li>
-                    % endfor
-                </ol>
-            </li>
-        % endfor
-    </ol>
+                                        </div> <!-- /details -->
+                                    </div> <!-- /subsection-text -->
+                                    <div class="subsection-actions">
+                                        ## Resume button (if last visited section)
+                                        % if subsection['current']:
+                                            <span class="sr-only">${ _("This is your last visited course section.") }</span>
+                                            <span class="resume-right">
+                                                <b>${ _("Resume Course") }</b>
+                                                <span class="icon fa fa-arrow-circle-right" aria-hidden="true"></span>
+                                            </span>
+                                        %endif
+                                    </div>
+                                </a>
+                            </li>
+                        % endfor
+                    </ol>
+                </li>
+            % endfor
+        </ol>
+    % else:
+        <div class="well depth-0 message-area">
+            <div class="copy-large">
+                <span class="icon fa fa-calendar-o" aria-hidden="true"></span>
+                <%
+                course_started = course.start.date() <= date.today()
+                %>
+
+                % if course.start_date_is_still_default:
+                    ${_("This course has not started yet.")}
+                % elif course_started:
+                    ${_("This course has no content.")}
```

Comment:
I like @robrap's suggestion of something that is more learner-friendly just in case. For the case where there is a start date but it's in the past, how about "We're still working on course content." without any mention of the course start? 
The message for other scenarios look good to me.